### PR TITLE
Basic registry credentials support

### PR DESCRIPTION
**Pull request by FrimIdan**

### CODE REVIEW

File: pkg/utils/k8s/k8s_utils_test.go [hunk 1]
```diff
@@ -41,3 +48,160 @@ func TestParseImageHash(t *testing.T) {
...
+			name: "match not specific image name",
+			args: args{
+				secrets: []*corev1.Secret{
+					{
+						TypeMeta: metav1.TypeMeta{
+							Kind:       "Secret",
+							APIVersion: "v1",
+						},
+						ObjectMeta: metav1.ObjectMeta{
+							Name:      "no-match",
```
Comment (akpsgit) [inline pkg/utils/k8s/k8s_utils_test.go:127]: no match?

File: pkg/scanner/creds/basic.go [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+func CreateBasicRegcred(clientset kubernetes.Interface, secretNamespace string) *BasicRegcred {
+	return &BasicRegcred{
+		credsCommon: credsCommon{
+			clientset:       clientset,
+			secretNamespace: secretNamespace,
+		},
+	}
+}
+
+func (u BasicRegcred) ShouldAdd() bool {
```
Comment (akpsgit) [inline pkg/scanner/creds/basic.go:30]: why not pointer?

File: pkg/scanner/creds/basic.go [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+	if u.isSecretExists == nil {
+		found := isSecretExists(u.clientset, BasicRegcredSecretName, u.secretNamespace)
+		u.isSecretExists = &found
+	}
+
+	return *u.isSecretExists
+}
+
+func (u BasicRegcred) Add(job *batchv1.Job) {
```
Comment (akpsgit) [inline pkg/scanner/creds/basic.go:39]: pointer

File: pkg/scanner/creds/basic.go [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+	batchv1 "k8s.io/api/batch/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+const (
+	BasicRegcredSecretName = "basic-regcred"
+)
+
+type BasicRegcred struct {
```
Comment (akpsgit) [inline pkg/scanner/creds/basic.go:14]: Regcred -> RegCred?

File: pkg/utils/k8s/k8s_utils_test.go (hunk 1 above)
Comment (FrimIdan) [reply pkg/utils/k8s/k8s_utils_test.go:127]: Didn't understand the question.

Comment (FrimIdan) [reply pkg/utils/k8s/k8s_utils_test.go:127]: There are 2 secrets here, the test will match the second one named `regcred`

**Review by akpsgit — APPROVED** [review]

**Pull request merged**